module/manage_requests.py

```diff
@@ -0,0 +1,75 @@
+import yaml
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup
+from telegram.ext import CallbackContext
+from module.create_connection import create_connection
+from module.add_item import add_item
+from module.find import find
+from module.shared import DB_PATH, error_message
+
+
+def add_request(context: CallbackContext, chat_id: int, isbn: str, title: str, authors: str, username: str, price: str) -> None:
+    item = (chat_id, isbn, title, authors, username, price)
+    sql = """ INSERT INTO Requests(ChatID, ISBN, Title, Authors, Seller, Price)
+              VALUES(?,?,?,?,?,?) """
+    conn = create_connection(DB_PATH)
+    if not conn:
+        context.bot.send_message(chat_id, error_message)
+        return
+    cur = conn.cursor()
+    cur.execute(sql, item)
+    conn.commit()
+    conn.close()
```

Comment:
this code is really similar to the code below under delete_request, you could wrap it into a beautiful function and reduce the redundancy of the code
this is up to you, you could make a function that manages connection, execution, commit and closing of the conn, or keep it as it is

May you could do this in another PR if you prefer